spec/factories/cart.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :cart do
+    content { {1 => 3, 2 => 3, 3 => 3} }
```

Comment:
do these keys need to be strings? I forgot if we changed our logic to allow them as integers or not.